### PR TITLE
Fixes #1236: Update Docs for get parameter

### LINKED ISSUE #1236: Update Docs for get parameter:- filter
- Issue type:Feature request

Update docs for get-parameter `filter` introduced in PR #1164 and #1190

### PR DESCRIPTION
**Pull request by vibhcool**
Fixes issue #1236
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by vibhcool** [comment]
Open for review :)

File: html/api.html [hunk 1]
```diff
@@ -348,77 +348,104 @@ <h2 class="sub-header">
...
+  "readme_2": "This is supposed to be the back-end of a search portal. For the api, see http://loklak.org/api.html",
+  "readme_3": "Parameters q=(query), source=(cache|backend|twitter|all), callback=p for jsonp, maximumRecords=(message count), minified=(true|false)",
+  "search_metadata": {
+    "startRecord": "1",
+    "maximumRecords": "1",
+    "count": "2",
+    "hits": 2,
+    "period": 81095001,
+    "query": "fossasia from:fossasia",
+    "filter": "video",
```
Comment (singhpratyush) [inline html/api.html]: What if opted for both video and image?

File: html/api.html [hunk 2]
```diff
@@ -348,77 +348,104 @@ <h2 class="sub-header">
   <li><div class="tabcol0">minified</div>
     <div class="tabcol2">&nbsp;=&nbsp;&lt;true|false&gt;</div>//
     minify the result, default false, i.e. minified=true</li>
+  <li><div class="tabcol0">filter</div>
+    <div class="tabcol2">&nbsp;=&nbsp;&lt;image|video|image,video&gt;</div>//
```
Comment (vibhcool) [inline html/api.html]: @singhpratyush , I have mentioned here, about different options

Comment (vibhcool) [inline html/api.html]: you can opt both video and image by using `filter=video,image`

File: html/api.html (hunk 1 above)
Comment (singhpratyush) [inline html/api.html]: I was asking about the JSON response in case if multiple options are selected.

Comment (vibhcool) [inline html/api.html]: json response's `filter` key will have same value as that of the query, it will be `"filter": "video, image"`

**Pull request merged**